src/unix/os390.c

```diff
@@ -576,6 +568,21 @@ int uv_fs_event_start(uv_fs_event_t* handle, uv_fs_event_cb cb,
 }
 
 
+int uv_fs_event_start(uv_fs_event_t* handle, uv_fs_event_cb cb,
+                      const char* filename, unsigned int flags) {
+  char* path;
+
+  if (uv__is_active(handle))
+    return UV_EINVAL;
+
+  path = uv__strdup(filename);
+  if (path == NULL)
+    return UV_ENOMEM;
+
+  return os390_regfileint(handle, cb, path, flags);
```

Comment:
might need to free `path` here if this fails (pre-existing memory leak bug)

and the handle start code (assignment of cb, path, etc) should be here also

Reply:
I moved the handle start code, cb, and path assignment. But I wasn't able to move rfis_rftok assignment since the value of __rfis_rftok is set/updated by the call to `__w_pioctl`.